pkg/catalogv2/system/system_test.go

```diff
@@ -269,7 +288,11 @@ func TestIsInstalled(t *testing.T) {
 			assert.Equal(t, test.expectedInstalled, installed)
 			assert.Equal(t, test.expectedVersion, version)
 			assert.Equal(t, test.expectedValues, values)
-			assert.Equal(t, test.expectedErr, err != nil)
+			if test.expectedErr {
```

Comment:
Could you add the same adjustment to the other test - TestIsInstalledExactVersion?

Reply:
done